blurb/blurb.py

```diff
@@ -486,6 +488,9 @@ def finish_entry():
                 # we'll complain about the *first* error
                 # we see in the blurb file, which is a
                 # better user experience.
+                if key == "gh-issue" and int(value) < lowest_possible_gh_issue_number:
+                    throw("The gh-issue number should be 32426 or above.")
```

Comment:
The force push reverted my last suggestion, so here it is again :)
```suggestion
                    throw("The gh-issue number must be 32426 or above, not a PR number.")
```

Tip: don't force push, we squash merge at the end anyway. 
(https://devguide.python.org/getting-started/pull-request-lifecycle/#quick-guide)